packages/opentelemetry-node/lib/metrics/host.js

```diff
@@ -36,19 +36,11 @@ function enableHostMetrics() {
 // Ref (metrics in Kibana): https://github.com/elastic/kibana/pull/174700
 /** @type {metrics.View[]} */
 const HOST_METRICS_VIEWS = [
-    // drop `system.network.*` (not in Kibana)
+    // TODO: As for now we're going to drop all system metrics
```

Comment:
Why is this a TODO? Isn't this change accomplishing this TODO?

Reply:
Probably the phrasing is wrong. What crossed my mind was the possibility to have other system metrics that are not exported by the collector. I'll remove it